wlutil/initramfsRoot/setupSystem.sh

```diff
@@ -0,0 +1,22 @@
+/bin/busybox --install -s
+
+# Load platform drivers
+sh /loadDrivers.sh
+
+# Mount the /proc and /sys filesystems.
+mount -t proc none /proc
+mount -t sysfs none /sys
+mount -t devtmpfs none /dev
+
+if [ -e /dev/generic-blkdev ]; then
```

Comment:
This means that the kconfig and bootargs don't have to change based on the platform anymore